helpers/sendToAuditLogs.js

```diff
@@ -0,0 +1,33 @@
+const Discord = require('discord.js');
+
+/**
+ * Sends a message to the audit-logs channel
+ * @param {any} interaction = Discord.js interaction
+ * @param {Object} configObj Embed config object
+ * @param {string} configObj.color Color of the embed
+ * @param {string} configObj.titleMsg Title msg for embed
+ * @param {string} [configObj.description] Description for embed
+ */
```

Comment:
This is okay. I do think JSDoc comment tends to make reading source code a little harder sometimes. Let's keep these to just functions that we're using across multiple files, since that's where the editor hover info will be most useful.

Reply:
I agree that we should limit their use to shared functions.